### DISCUSSION TITLE
doesn't work...no way. please help,  windows11, miniconda[Bug]

### DISCUSSION THREAD

**Discussion by Zerthura** [coqui-ai/TTS#2881] [General Q&A]
### Describe the bug

after many problems to install it, finally it installed...but can't use it.
why so many problems using it on windows? 
I saw people installing it in 2 min on linux.
I need it for work, it is making me crazy.
Please help me to make it works.
I must create batch file to make it tts many files, but actually I am unable to make it  tts one file.
I am wasting weeks trying to solve all conflicts about this software. 
I need help.

### To Reproduce

(TTS) C:\Users\Administrator>tts --text "Text for TTS" --out_path output/path/speech.wav
 > tts_models/en/ljspeech/tacotron2-DDC is already downloaded.
 > vocoder_models/en/ljspeech/hifigan_v2 is already downloaded.
 > Using model: Tacotron2
 > Setting up Audio Processor...
 | > sample_rate:22050
 | > resample:False
 | > num_mels:80
 | > log_func:np.log
 | > min_level_db:-100
 | > frame_shift_ms:None
 | > frame_length_ms:None
 | > ref_level_db:20
 | > fft_size:1024
 | > power:1.5
 | > preemphasis:0.0
 | > griffin_lim_iters:60
 | > signal_norm:False
 | > symmetric_norm:True
 | > mel_fmin:0
 | > mel_fmax:8000.0
 | > pitch_fmin:1.0
 | > pitch_fmax:640.0
 | > spec_gain:1.0
 | > stft_pad_mode:reflect
 | > max_norm:4.0
 | > clip_norm:True
 | > do_trim_silence:True
 | > trim_db:60
 | > do_sound_norm:False
 | > do_amp_to_db_linear:True
 | > do_amp_to_db_mel:True
 | > do_rms_norm:False
 | > db_level:None
 | > stats_path:None
 | > base:2.718281828459045
 | > hop_length:256
 | > win_length:1024
 > Model's reduction rate `r` is set to: 1
 > Vocoder Model: hifigan
 > Setting up Audio Processor...
 | > sample_rate:22050
 | > resample:False
 | > num_mels:80
 | > log_func:np.log
 | > min_level_db:-100
 | > frame_shift_ms:None
 | > frame_length_ms:None
 | > ref_level_db:20
 | > fft_size:1024
 | > power:1.5
 | > preemphasis:0.0
 | > griffin_lim_iters:60
 | > signal_norm:False
 | > symmetric_norm:True
 | > mel_fmin:0
 | > mel_fmax:8000.0
 | > pitch_fmin:1.0
 | > pitch_fmax:640.0
 | > spec_gain:1.0
 | > stft_pad_mode:reflect
 | > max_norm:4.0
 | > clip_norm:True
 | > do_trim_silence:False
 | > trim_db:60
 | > do_sound_norm:False
 | > do_amp_to_db_linear:True
 | > do_amp_to_db_mel:True
 | > do_rms_norm:False
 | > db_level:None
 | > stats_path:None
 | > base:2.718281828459045
 | > hop_length:256
 | > win_length:1024
 > Generator Model: hifigan_generator
 > Discriminator Model: hifigan_discriminator
Removing weight norm...
 > Text: Text for TTS
 > Text splitted to sentences.
['Text for TTS']
 > Processing time: 0.4076573848724365
 > Real-time factor: 0.23714767139186432
 > Saving output to output/path/speech.wav
Traceback (most recent call last):
  File "C:\Users\shula\miniconda3\envs\TTS\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\shula\miniconda3\envs\TTS\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\shula\miniconda3\envs\TTS\Scripts\tts.exe\__main__.py", line 7, in <module>
  File "C:\Users\shula\miniconda3\envs\TTS\lib\site-packages\TTS\bin\synthesize.py", line 451, in main
    synthesizer.save_wav(wav, args.out_path)
  File "C:\Users\shula\miniconda3\envs\TTS\lib\site-packages\TTS\utils\synthesizer.py", line 244, in save_wav
    save_wav(wav=wav, path=path, sample_rate=self.output_sample_rate)
  File "C:\Users\shula\miniconda3\envs\TTS\lib\site-packages\TTS\utils\audio\numpy_transforms.py", line 439, in save_wav
    scipy.io.wavfile.write(path, sample_rate, wav_norm.astype(np.int16))
  File "C:\Users\shula\miniconda3\envs\TTS\lib\site-packages\scipy\io\wavfile.py", line 767, in write
    fid = open(filename, 'wb')
FileNotFoundError: [Errno 2] No such file or directory: 'output/path/speech.wav'

### Expected behavior

i made the most simple test, it failed

### Environment

```shell
(TTS) C:\Users\Administrator>conda list
# packages in environment at C:\Users\shula\miniconda3\envs\TTS:
#
# Name                    Version                   Build  Channel
absl-py                   1.4.0                    pypi_0    pypi
accelerate                0.21.0                   pypi_0    pypi
aiohttp                   3.8.5                    pypi_0    pypi
aiosignal                 1.3.1                    pypi_0    pypi
anyascii                  0.3.2                    pypi_0    pypi
appdirs                   1.4.4                    pypi_0    pypi
async-timeout             4.0.3                    pypi_0    pypi
attrs                     23.1.0                   pypi_0    pypi
audioread                 3.0.0                    pypi_0    pypi
babel                     2.12.1                   pypi_0    pypi
bangla                    0.0.2                    pypi_0    pypi
blinker                   1.6.2                    pypi_0    pypi
bnnumerizer               0.0.2                    pypi_0    pypi
bnunicodenormalizer       0.1.1                    pypi_0    pypi
boltons                   23.0.0                   pypi_0    pypi
bzip2                     1.0.8                he774522_0
ca-certificates           2023.05.30           haa95532_0
cachetools                5.3.1                    pypi_0    pypi
certifi                   2023.7.22                pypi_0    pypi
cffi                      1.15.1                   pypi_0    pypi
charset-normalizer        3.2.0                    pypi_0    pypi
clean-fid                 0.1.35                   pypi_0    pypi
click                     8.1.6                    pypi_0    pypi
clip-anytorch             2.5.2                    pypi_0    pypi
colorama                  0.4.6                    pypi_0    pypi
contourpy                 1.1.0                    pypi_0    pypi
coqpit                    0.0.17                   pypi_0    pypi
cycler                    0.11.0                   pypi_0    pypi
cython                    0.29.30                  pypi_0    pypi
dateparser                1.1.8                    pypi_0    pypi
decorator                 5.1.1                    pypi_0    pypi
docker-pycreds            0.4.0                    pypi_0    pypi
docopt                    0.6.2                    pypi_0    pypi
einops                    0.6.1                    pypi_0    pypi
encodec                   0.1.1                    pypi_0    pypi
filelock                  3.12.2                   pypi_0    pypi
flask                     2.3.2                    pypi_0    pypi
fonttools                 4.42.0                   pypi_0    pypi
frozenlist                1.4.0                    pypi_0    pypi
fsspec                    2023.6.0                 pypi_0    pypi
ftfy                      6.1.1                    pypi_0    pypi
g2pkk                     0.1.2                    pypi_0    pypi
gitdb                     4.0.10                   pypi_0    pypi
gitpython                 3.1.32                   pypi_0    pypi
google-auth               2.22.0                   pypi_0    pypi
google-auth-oauthlib      1.0.0                    pypi_0    pypi
grpcio                    1.57.0                   pypi_0    pypi
gruut                     2.2.3                    pypi_0    pypi
gruut-ipa                 0.13.0                   pypi_0    pypi
gruut-lang-de             2.0.0                    pypi_0    pypi
gruut-lang-en             2.0.0                    pypi_0    pypi
gruut-lang-es             2.0.0                    pypi_0    pypi
gruut-lang-fr             2.0.2                    pypi_0    pypi
huggingface-hub           0.16.4                   pypi_0    pypi
idna                      3.4                      pypi_0    pypi
imageio                   2.31.1                   pypi_0    pypi
inflect                   5.6.0                    pypi_0    pypi
itsdangerous              2.1.2                    pypi_0    pypi
jamo                      0.4.1                    pypi_0    pypi
jieba                     0.42.1                   pypi_0    pypi
jinja2                    3.1.2                    pypi_0    pypi
joblib                    1.3.2                    pypi_0    pypi
jsonlines                 1.2.0                    pypi_0    pypi
jsonmerge                 1.9.2                    pypi_0    pypi
jsonschema                4.19.0                   pypi_0    pypi
jsonschema-specifications 2023.7.1                 pypi_0    pypi
k-diffusion               0.0.15                   pypi_0    pypi
kiwisolver                1.4.4                    pypi_0    pypi
kornia                    0.7.0                    pypi_0    pypi
lazy-loader               0.3                      pypi_0    pypi
libffi                    3.4.4                hd77b12b_0
librosa                   0.10.0                   pypi_0    pypi
llvmlite                  0.40.1                   pypi_0    pypi
markdown                  3.4.4                    pypi_0    pypi
markupsafe                2.1.3                    pypi_0    pypi
matplotlib                3.7.2                    pypi_0    pypi
mpmath                    1.3.0                    pypi_0    pypi
msgpack                   1.0.5                    pypi_0    pypi
multidict                 6.0.4                    pypi_0    pypi
networkx                  2.8.8                    pypi_0    pypi
nltk                      3.8.1                    pypi_0    pypi
num2words                 0.5.12                   pypi_0    pypi
numba                     0.57.0                   pypi_0    pypi
numpy                     1.23.0                   pypi_0    pypi
oauthlib                  3.2.2                    pypi_0    pypi
openssl                   3.0.10               h2bbff1b_0
packaging                 23.1                     pypi_0    pypi
pandas                    2.0.3                    pypi_0    pypi
pathtools                 0.1.2                    pypi_0    pypi
pillow                    10.0.0                   pypi_0    pypi
pip                       23.2.1          py310haa95532_0
platformdirs              3.10.0                   pypi_0    pypi
pooch                     1.7.0                    pypi_0    pypi
protobuf                  4.24.0                   pypi_0    pypi
psutil                    5.9.5                    pypi_0    pypi
pyasn1                    0.5.0                    pypi_0    pypi
pyasn1-modules            0.3.0                    pypi_0    pypi
pycparser                 2.21                     pypi_0    pypi
pynndescent               0.5.10                   pypi_0    pypi
pyparsing                 3.0.9                    pypi_0    pypi
pypinyin                  0.49.0                   pypi_0    pypi
pysbd                     0.3.4                    pypi_0    pypi
python                    3.10.12              he1021f5_0
python-crfsuite           0.9.9                    pypi_0    pypi
python-dateutil           2.8.2                    pypi_0    pypi
pytz                      2023.3                   pypi_0    pypi
pywavelets                1.4.1                    pypi_0    pypi
pyyaml                    6.0.1                    pypi_0    pypi
referencing               0.30.2                   pypi_0    pypi
regex                     2023.8.8                 pypi_0    pypi
requests                  2.31.0                   pypi_0    pypi
requests-oauthlib         1.3.1                    pypi_0    pypi
resize-right              0.0.2                    pypi_0    pypi
rpds-py                   0.9.2                    pypi_0    pypi
rsa                       4.9                      pypi_0    pypi
safetensors               0.3.2                    pypi_0    pypi
scikit-image              0.21.0                   pypi_0    pypi
scikit-learn              1.3.0                    pypi_0    pypi
scipy                     1.11.1                   pypi_0    pypi
sentry-sdk                1.29.2                   pypi_0    pypi
setproctitle              1.3.2                    pypi_0    pypi
setuptools                68.0.0          py310haa95532_0
six                       1.16.0                   pypi_0    pypi
smmap                     5.0.0                    pypi_0    pypi
soundfile                 0.12.1                   pypi_0    pypi
soxr                      0.3.5                    pypi_0    pypi
sqlite                    3.41.2               h2bbff1b_0
sympy                     1.12                     pypi_0    pypi
tensorboard               2.14.0                   pypi_0    pypi
tensorboard-data-server   0.7.1                    pypi_0    pypi
threadpoolctl             3.2.0                    pypi_0    pypi
tifffile                  2023.7.18                pypi_0    pypi
tk                        8.6.12               h2bbff1b_0
tokenizers                0.13.3                   pypi_0    pypi
torch                     2.0.1                    pypi_0    pypi
torchaudio                2.0.2                    pypi_0    pypi
torchdiffeq               0.2.3                    pypi_0    pypi
torchsde                  0.2.5                    pypi_0    pypi
torchvision               0.15.2                   pypi_0    pypi
tqdm                      4.66.1                   pypi_0    pypi
trainer                   0.0.30                   pypi_0    pypi
trampoline                0.1.2                    pypi_0    pypi
transformers              4.31.0                   pypi_0    pypi
tts                       0.16.1                   pypi_0    pypi
typing-extensions         4.7.1                    pypi_0    pypi
tzdata                    2023.3                   pypi_0    pypi
tzlocal                   5.0.1                    pypi_0    pypi
umap-learn                0.5.1                    pypi_0    pypi
urllib3                   1.26.16                  pypi_0    pypi
vc                        14.2                 h21ff451_1
vs2015_runtime            14.27.29016          h5e58377_2
wandb                     0.15.8                   pypi_0    pypi
wcwidth                   0.2.6                    pypi_0    pypi
werkzeug                  2.3.6                    pypi_0    pypi
wheel                     0.38.4          py310haa95532_0
xz                        5.4.2                h8cc25b3_0
yarl                      1.9.2                    pypi_0    pypi
zlib                      1.2.13               h8cc25b3_0
```

### Additional context

DEPRECATION: torchsde 0.2.5 has a non-standard dependency specifier numpy>=1.19.*; python_version >= "3.7". pip 23.3 will enforce this behaviour change. A possible replacement is to upgrade to a newer version of torchsde or contact the author to suggest that they release a version with a conforming dependency specifiers. Discussion can be found at https://github.com/pypa/pip/issues/12063
Installing collected packages: mecab-python3
Successfully installed mecab-python3-1.0.6

(TTS) C:\Users\Administrator>pip install unidic-lite
Collecting unidic-lite
  Using cached unidic-lite-1.0.8.tar.gz (47.4 MB)
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: unidic-lite
  Building wheel for unidic-lite (setup.py) ... done
  Created wheel for unidic-lite: filename=unidic_lite-1.0.8-py3-none-any.whl size=47658833 sha256=57d389c236768b40af598139cab9147bb1ceca437eac9c206ed9a8c85b85811f
  Stored in directory: c:\users\administrator\appdata\local\pip\cache\wheels\89\e8\68\f9ac36b8cc6c8b3c96888cd57434abed96595d444f42243853
Successfully built unidic-lite
DEPRECATION: torchsde 0.2.5 has a non-standard dependency specifier numpy>=1.19.*; python_version >= "3.7". pip 23.3 will enforce this behaviour change. A possible replacement is to upgrade to a newer version of torchsde or contact the author to suggest that they release a version with a conforming dependency specifiers. Discussion can be found at https://github.com/pypa/pip/issues/12063
Installing collected packages: unidic-lite
Successfully installed unidic-lite-1.0.8

(TTS) C:\Users\Administrator>tts --text "Text for TTS" --out_path output/path/speech.wav
 > Downloading model to C:\Users\Administrator\AppData\Local\tts\tts_models--en--ljspeech--tacotron2-DDC
100%|██████████████████████████████████████████████████████████████████████████████| 113M/113M [01:25<00:00, 1.31MiB/s]
 > Model's license - apache 2.0
 > Check https://choosealicense.com/licenses/apache-2.0/ for more info.
 > Downloading model to C:\Users\Administrator\AppData\Local\tts\vocoder_models--en--ljspeech--hifigan_v2
100%|█████████████████████████████████████████████████████████████████████████████| 3.80M/3.80M [00:05<00:00, 712kiB/s]
 > Model's license - apache 2.0
 > Check https://choosealicense.com/licenses/apache-2.0/ for more info.
 > Using model: Tacotron2
 > Setting up Audio Processor...
 | > sample_rate:22050
 | > resample:False
 | > num_mels:80
 | > log_func:np.log
 | > min_level_db:-100
 | > frame_shift_ms:None
 | > frame_length_ms:None
 | > ref_level_db:20
 | > fft_size:1024
 | > power:1.5
 | > preemphasis:0.0
 | > griffin_lim_iters:60
 | > signal_norm:False
 | > symmetric_norm:True
 | > mel_fmin:0
 | > mel_fmax:8000.0
 | > pitch_fmin:1.0
 | > pitch_fmax:640.0
 | > spec_gain:1.0
 | > stft_pad_mode:reflect
 | > max_norm:4.0
 | > clip_norm:True
 | > do_trim_silence:True
 | > trim_db:60
 | > do_sound_norm:False
 | > do_amp_to_db_linear:True
 | > do_amp_to_db_mel:True
 | > do_rms_norm:False
 | > db_level:None
 | > stats_path:None
 | > base:2.718281828459045
 | > hop_length:256
 | > win_length:1024
 > Model's reduction rate `r` is set to: 1
 > Vocoder Model: hifigan
 > Setting up Audio Processor...
 | > sample_rate:22050
 | > resample:False
 | > num_mels:80
 | > log_func:np.log
 | > min_level_db:-100
 | > frame_shift_ms:None
 | > frame_length_ms:None
 | > ref_level_db:20
 | > fft_size:1024
 | > power:1.5
 | > preemphasis:0.0
 | > griffin_lim_iters:60
 | > signal_norm:False
 | > symmetric_norm:True
 | > mel_fmin:0
 | > mel_fmax:8000.0
 | > pitch_fmin:1.0
 | > pitch_fmax:640.0
 | > spec_gain:1.0
 | > stft_pad_mode:reflect
 | > max_norm:4.0
 | > clip_norm:True
 | > do_trim_silence:False
 | > trim_db:60
 | > do_sound_norm:False
 | > do_amp_to_db_linear:True
 | > do_amp_to_db_mel:True
 | > do_rms_norm:False
 | > db_level:None
 | > stats_path:None
 | > base:2.718281828459045
 | > hop_length:256
 | > win_length:1024
 > Generator Model: hifigan_generator
 > Discriminator Model: hifigan_discriminator
Removing weight norm...
 > Text: Text for TTS
 > Text splitted to sentences.
['Text for TTS']
 > Processing time: 0.5023317337036133
 > Real-time factor: 0.298266230293103
 > Saving output to output/path/speech.wav
Traceback (most recent call last):
  File "C:\Users\shula\miniconda3\envs\TTS\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\shula\miniconda3\envs\TTS\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\shula\miniconda3\envs\TTS\Scripts\tts.exe\__main__.py", line 7, in <module>
  File "C:\Users\shula\miniconda3\envs\TTS\lib\site-packages\TTS\bin\synthesize.py", line 451, in main
    synthesizer.save_wav(wav, args.out_path)
  File "C:\Users\shula\miniconda3\envs\TTS\lib\site-packages\TTS\utils\synthesizer.py", line 244, in save_wav
    save_wav(wav=wav, path=path, sample_rate=self.output_sample_rate)
  File "C:\Users\shula\miniconda3\envs\TTS\lib\site-packages\TTS\utils\audio\numpy_transforms.py", line 439, in save_wav    scipy.io.wavfile.write(path, sample_rate, wav_norm.astype(np.int16))
  File "C:\Users\shula\miniconda3\envs\TTS\lib\site-packages\scipy\io\wavfile.py", line 767, in write
    fid = open(filename, 'wb')
FileNotFoundError: [Errno 2] No such file or directory: 'output/path/speech.wav'

(TTS) C:\Users\Administrator>tts --text "Text for TTS" --out_path output/path/speech.wav
 > tts_models/en/ljspeech/tacotron2-DDC is already downloaded.
 > vocoder_models/en/ljspeech/hifigan_v2 is already downloaded.
 > Using model: Tacotron2
 > Setting up Audio Processor...
 | > sample_rate:22050
 | > resample:False
 | > num_mels:80
 | > log_func:np.log
 | > min_level_db:-100
 | > frame_shift_ms:None
 | > frame_length_ms:None
 | > ref_level_db:20
 | > fft_size:1024
 | > power:1.5
 | > preemphasis:0.0
 | > griffin_lim_iters:60
 | > signal_norm:False
 | > symmetric_norm:True
 | > mel_fmin:0
 | > mel_fmax:8000.0
 | > pitch_fmin:1.0
 | > pitch_fmax:640.0
 | > spec_gain:1.0
 | > stft_pad_mode:reflect
 | > max_norm:4.0
 | > clip_norm:True
 | > do_trim_silence:True
 | > trim_db:60
 | > do_sound_norm:False
 | > do_amp_to_db_linear:True
 | > do_amp_to_db_mel:True
 | > do_rms_norm:False
 | > db_level:None
 | > stats_path:None
 | > base:2.718281828459045
 | > hop_length:256
 | > win_length:1024
 > Model's reduction rate `r` is set to: 1
 > Vocoder Model: hifigan
 > Setting up Audio Processor...
 | > sample_rate:22050
 | > resample:False
 | > num_mels:80
 | > log_func:np.log
 | > min_level_db:-100
 | > frame_shift_ms:None
 | > frame_length_ms:None
 | > ref_level_db:20
 | > fft_size:1024
 | > power:1.5
 | > preemphasis:0.0
 | > griffin_lim_iters:60
 | > signal_norm:False
 | > symmetric_norm:True
 | > mel_fmin:0
 | > mel_fmax:8000.0
 | > pitch_fmin:1.0
 | > pitch_fmax:640.0
 | > spec_gain:1.0
 | > stft_pad_mode:reflect
 | > max_norm:4.0
 | > clip_norm:True
 | > do_trim_silence:False
 | > trim_db:60
 | > do_sound_norm:False
 | > do_amp_to_db_linear:True
 | > do_amp_to_db_mel:True
 | > do_rms_norm:False
 | > db_level:None
 | > stats_path:None
 | > base:2.718281828459045
 | > hop_length:256
 | > win_length:1024
 > Generator Model: hifigan_generator
 > Discriminator Model: hifigan_discriminator
Removing weight norm...
 > Text: Text for TTS
 > Text splitted to sentences.
['Text for TTS']
 > Processing time: 0.4537527561187744
 > Real-time factor: 0.2712919813562629
 > Saving output to output/path/speech.wav
Traceback (most recent call last):
  File "C:\Users\shula\miniconda3\envs\TTS\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\shula\miniconda3\envs\TTS\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\shula\miniconda3\envs\TTS\Scripts\tts.exe\__main__.py", line 7, in <module>
  File "C:\Users\shula\miniconda3\envs\TTS\lib\site-packages\TTS\bin\synthesize.py", line 451, in main
    synthesizer.save_wav(wav, args.out_path)
  File "C:\Users\shula\miniconda3\envs\TTS\lib\site-packages\TTS\utils\synthesizer.py", line 244, in save_wav
    save_wav(wav=wav, path=path, sample_rate=self.output_sample_rate)
  File "C:\Users\shula\miniconda3\envs\TTS\lib\site-packages\TTS\utils\audio\numpy_transforms.py", line 439, in save_wav    scipy.io.wavfile.write(path, sample_rate, wav_norm.astype(np.int16))
  File "C:\Users\shula\miniconda3\envs\TTS\lib\site-packages\scipy\io\wavfile.py", line 767, in write
    fid = open(filename, 'wb')
FileNotFoundError: [Errno 2] No such file or directory: 'output/path/speech.wav'

**Comment by kin0303** [comment]
Because you don't have this path: ```'output/path/```
